### ISSUE TITLE
Schoology updated icons revert to old versions

### ISSUE THREAD

**Issue by RobotChickens** [aopell/SchoologyPlus#325]
### Describe the bug
> Our domain got access to a new UI update for Schoology. This update includes new icons for folders, to-do, and others. However, these icons only show for a second before being reverted to the old ones.

### Steps to reproduce
> All I do is open up Schoology **without Schoology+ installed**. Clearing my cookies and cache doesn't do anything.

### Expected behavior
> I expect the updated Icons to stay, but they keep changing back. I also thought this extension would fix it, but it didn't.

### Screenshots
>New icons: ![image](https://github.com/aopell/SchoologyPlus/assets/112828029/2e6f83c8-4254-439b-a8d7-fb1d31013c2c)
Old icons: 
![image](https://github.com/aopell/SchoologyPlus/assets/112828029/5310828f-4ccc-4a14-bdae-ca73cc3701a8)

### Platform
> Google Chrome Version 121.0.6167.16, lms.fcps.org,  Schoology Plus v7.8.6

**Comment by aopell** [comment]
Hi @RobotChickens, could you clarify if the issue is happening only when Schoology Plus occurs?

I can't tell if you're saying that the issue is happening only with Schoology Plus or if it's happening always, but you just hoped it could be fixed by Schoology Plus

**Comment by RobotChickens** [comment]
> Hi @RobotChickens, could you clarify if the issue is happening only when Schoology Plus occurs?
> 
> I can't tell if you're saying that the issue is happening only with Schoology Plus or if it's happening always, but you just hoped it could be fixed by Schoology Plus

Yes, it happens all the time _without_ the extension. I installed it thinking that the "update Icons" feature would help, but that ended up being something else. The extension does not change the behavior of the icons reverting.
